### PR TITLE
[plugin/statistics] Synchronize Current book with footer

### PR DESCRIPTION
**Pull request by robert00s**
Better synchronize statistics `Current Book` with footer.
Also show `Current page/Total pages` instead of `Read pages/Total pages`.
Add `Read pages` row.

Before:

After:

### CODE REVIEW

File: plugins/statistics.koplugin/main.lua [hunk 1]
```diff
@@ -1016,11 +1016,11 @@ function ReaderStatistics:getCurrentStat(id_book)
         { _("Total time"), util.secondsToClock(total_time_book, false) },
         { _("Total highlights"), tonumber(highlights) },
         { _("Total notes"), tonumber(notes) },
+        { _("Total read pages"), tonumber(total_read_pages) },
```
Comment (Frenzie) [inline plugins/statistics.koplugin/main.lua]: Total pages read ought to sound better.
```suggestion
        { _("Total pages read"), tonumber(total_read_pages) },
```

Comment (NiLuJe) [reply plugins/statistics.koplugin/main.lua]: Which made me realize that `Current pages` sounds iffy, too. `Current pages read`?

Comment (NiLuJe) [reply plugins/statistics.koplugin/main.lua]: I'd also possibly swap "period" for "session", and repeat it where appropriate (i.e., "Pages read this session", "Current session duration", ...).

Comment (NiLuJe) [reply plugins/statistics.koplugin/main.lua]: "Time spent reading today", "Pages read today"

**Comment by robert00s** [comment]
Thanks. Done :)

**Comment by Frenzie** [comment]
(Merging because I need to test if the OTA file is generated correctly and this is a small one.)

**Pull request merged**